app/src/main/java/com/sun/android/CatDetailFragment.kt

```diff
@@ -0,0 +1,198 @@
+package com.sun.android
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.sun.android.databinding.FragmentCatDetailBinding
+
+private const val CAT_INDEX_KEY = "catIndex"
+private const val LANGUAGE_KEY = "language"
+private const val DEFAULT_CAT_INDEX = 1
+private const val VIETNAMESE_LANGUAGE = "vi"
+private const val ENGLISH_LANGUAGE = "en"
+private const val JAPANESE_LANGUAGE = "jp"
+private const val DEFAULT_LANGUAGE = "vi"
+private const val CAT_1_INDEX = 1
+private const val CAT_2_INDEX = 2
+private const val CAT_3_INDEX = 3
+private const val CAT_4_INDEX = 4
+private const val CAT_5_INDEX = 5
+private const val NOT_FOUND = -1
+
+class CatDetailFragment : Fragment() {
+
+    private lateinit var binding: FragmentCatDetailBinding
+    private var catIndex = DEFAULT_CAT_INDEX
+    private var language = DEFAULT_LANGUAGE
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.apply {
+            catIndex = this.getInt(CAT_INDEX_KEY, DEFAULT_CAT_INDEX)
+            language = this.getString(LANGUAGE_KEY, DEFAULT_LANGUAGE)
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentCatDetailBinding.inflate(layoutInflater)
+
+        if (catIndex != -1) {
+            val imageCatResId: Int = when (catIndex) {
+                CAT_1_INDEX -> {
+                    resources.getIdentifier(
+                        "meo_xiem",
+                        "drawable",
+                        requireContext().packageName
+                    )
+                }
```

Comment:
```suggestion
                CAT_1_INDEX -> R.drawable.meo_xiem
```
tương tự cho những chỗ khác nữa nhé

Reply:
Đã sửa